cmd/truss/main.go

```diff
@@ -325,33 +335,30 @@ func readPreviousGeneration(serviceDir string) ([]truss.NamedReadWriter, error)
 }
 
 // simpleFileConstructor has function makeSimpleFile of type filepath.WalkFunc
-// This allows for filepath.Walk to be called with makeSimpleFile and build a truss.SimpleFile
-// for all files in a direcotry
+// This allows for filepath.Walk to be called with makeSimpleFile and build a fileSystem
+// for all files in a directory
 type simpleFileConstructor struct {
 	dir   string
-	files []truss.NamedReadWriter
+	files map[string]io.Reader
```

Comment:
Here you've used the underlying definition of a `fileSystem` and I don't think anything's suffered for it.